core/actions/data_preparation.ts

```diff
@@ -36,7 +36,7 @@ export class DataPreparation extends ActionBuilder<dataform.DataPreparation> {
     config.filename = resolveActionsConfigFilename(config.filename, configPath);
     const dataPreparationContents = nativeRequire(config.filename).asJson;
     const dataPreparationDefinition = parseDataPreparationDefinitionJson(dataPreparationContents);
```

Comment:
Nit: the term `dataPreparationContents` is overloaded now, so should be renamed - something like `dataPreparationAsJson = ...`

Reply:
Updated